specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-02-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+          {
+            "$ref": "#/parameters/EntityFilterParameter"
+          },
+          {
+            "$ref": "#/parameters/EntityViewParameter"
+          },
+          {
+            "$ref": "#/parameters/GroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CacheControlHeader"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/EntityListResult"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "GetEntities": {
+            "$ref": "./examples/GetEntities.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/startTenantBackfill": {
+      "post": {
+        "tags": [
+          "TenantBackfill"
+        ],
+        "operationId": "StartTenantBackfill",
+        "description": "Starts backfilling subscriptions for the Tenant.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/TenantBackfillStatusResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "StartTenantBackfill": {
+            "$ref": "./examples/StartTenantBackfillRequest.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/tenantBackfillStatus": {
+      "post": {
+        "tags": [
+          "TenantBackfill"
+        ],
+        "operationId": "TenantBackfillStatus",
+        "description": "Gets tenant backfill status",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/TenantBackfillStatusResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "TenantBackfillStatus": {
+            "$ref": "./examples/TenantBackfillStatusRequest.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/applications/{applicationId}": {
```

Comment:
>/providers/Microsoft.Management/applications/{applicationId} [](http://example.com/codeflow?start=5&length=60)

New tenant-level APIs require PAS signoff. This is done through email. If you haven't already, send email to SARBACRev@microsoft.com, prasnal@microsoft.com, ARM Auth Team <armrbac@microsoft.com> and request signoff. If you've already done this, just forward the email to me.